services/admin/backend/cmd/main.go

```diff
@@ -71,47 +52,73 @@ func main() {
 		log.Fatal("Unexpected failure connecting: ", err.Error())
 	}
 
-	projectRepository := projectRepository.NewProjectRepository(client)
+	projectRepo := projectRepository.NewProjectRepository(client)
 
-	projectService := project.NewService(projectRepository)
+	projectService := project.NewService(projectRepo)
 
-	metricService, err := metric.NewPrometheusService()
-	if err != nil {
-		log.Fatal(err.Error())
-	}
-	r := mux.NewRouter()
-	//handlers
-	n := negroni.New(
-		negroni.HandlerFunc(middleware.Cors),
-		negroni.HandlerFunc(middleware.Metrics(metricService)),
-		negroni.NewLogger(),
-	)
-
-	//organization
-	handler.MakeOrganizationHandlers(r, *n, organizationService)
-
-	//project
-	handler.MakeProjectHandlers(r, *n, projectService)
-
-	http.Handle("/", r)
-	http.Handle("/metrics", promhttp.Handler())
-	r.HandleFunc("/ping", func(w http.ResponseWriter, r *http.Request) {
-		w.WriteHeader(http.StatusOK)
-	})
-
-	logger := log.New(os.Stderr, "logger: ", log.Lshortfile)
-	srv := &http.Server{
-		ReadTimeout:  5 * time.Second,
-		WriteTimeout: 10 * time.Second,
-		Addr:         ":" + strconv.Itoa(8080),
-		// FIXME: get rid of deprecated github.com/gorilla/context library
-		Handler:  context.ClearHandler(http.DefaultServeMux),
-		ErrorLog: logger,
-	}
-	err = srv.ListenAndServe()
-	if err != nil {
-		log.Fatal(err.Error())
-	}
+	handler.MakeProjectHandlers(&s.Router, projectService)
+
+	log.Fatal(s.ListenAndServe())
+
+	// organizationRepository := repository.NewInmemDB()
```

Comment:
those commented out lines could be deleted too, right?

Reply:
removed in b681dfc